app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -607,4 +617,12 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
       this.intl.t('warning-title')
     );
   }
+
+  @action
+  showAgendaHasChangedToast() {
+    this.toaster.error(
+      this.intl.t('an-agenda-was-approved-since-modal-was-opened'),
+      this.intl.t('warning-title')
+    )
+  }
```

Comment:
agenda approve / agenda approve and close already have a try catch with an error toast. We could use those